Tests/Runtime/ClientSendTests.cs

```diff
@@ -123,32 +123,6 @@ public IEnumerator PiiTests_ShouldChangeApplicationDataPath_ApplicationDataPathD
             yield return null;
         }
 
-        [UnityTest]
-        public IEnumerator PiiTests_ShouldChangeSourceCodeIntegration_SourceCodeTextDoesntHaveUserNameAnymore()
```

Comment:
because this code provides tests for "sourceCode" integration responsible for displaying unity engine logs in the web debugger. Since we have breadcrumbs support now, we don't need to have it anymore.